index.d.ts

```diff
@@ -1,23 +1,22 @@
 /// <reference lib="esnext"/>
 
-export type AddRemoveListener<FirstArgumentType, RestArgumentsType> = (
+export type AddRemoveListener<Args extends any[]> = (
```

Comment:
I prefer using `unknown` over `any`.

`Args` => `Arguments`.

Reply:
Done and done!

Reply:
For what it's worth, when using rest types, either top type (`any` or `unknown`) is equivalent and has no tighter or weaker bound than the other.

It might be confusing to other TypeScript users to see `extends unknown[]` because that isn't the typical pattern, but it's indistinguishable at the type level from `extends any[]`, because *all* arrays extend both.

Reply:
> or what it's worth, when using rest types, either top type (any or unknown) is equivalent and has no tighter or weaker bounds than the other.

Yup, but I use `unknown` for everything else, so I prefer the consistency.

> It might be confusing to other TypeScript users to see extends unknown[] because that isn't the typical pattern

I expect users to know how `unknown` works.

Reply:
👍